portal-web/test/functional/com/liferay/portalweb/macros/journal/WebContent.macro

```diff
@@ -2425,12 +2425,10 @@ definition {
 				webContentTitle = "${webContentTitle}",
 				webContentTitleEscapeCharacter = "${webContentTitleEscapeCharacter}");
 
-			if (!(isSet(webContentLocaleFieldLabel))) {
-				var webContentLocaleFieldLabel = "en_US";
+			if (isSet(webContentLocaleFieldLabel)) {
```

Comment:
Hey @ViniLopes20 , next time please amend this fix on your commit where you add this condition: 37bda75d29752697660cdfd8001bd53dba1565c8